plottable.js

```diff
@@ -5029,8 +5029,8 @@ var Plottable;
                  */
                 get: function () {
                     var wrapper = new SVGTypewriter.Wrappers.Wrapper();
-                    if (this._tickLabelMaxWidth != null) {
```

Comment:
No, we now don't have any defaults. I felt it was simpler from an API perspective